docs/README.md

```diff
@@ -1,33 +0,0 @@
-# Welcome to MkDocs
-
-Install:
- - MkDocs - https://www.mkdocs.org/#installing-mkdocs
- - Material Theme - https://squidfunk.github.io/mkdocs-material/getting-started/#installation
- - mkdocs-redirects - https://github.com/mkdocs/mkdocs-redirects
-
-```sh
-pip install mkdocs mkdocs-material mkdocs-redirects
-```
-
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
-
-## Commands
-
-* `mkdocs serve` - Start the live-reloading docs server.
-* `mkdocs build` - Build the documentation site.
-* `mkdocs -h` - Print help message and exit.
```

Comment:
I tend to forget how to get things running locally. Can we keep it in some form (maybe needs update regarding installing dependencies)?